app/src/main/res/values-land/styles.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <style name="PlaceholderTextViewStyle">
         <item name="android:textSize">@dimen/text_size_placeholders</item>
-        <item name="fontPath">fonts/Roboto-Light.ttf</item>
+        <item name="android:fontFamily" tools:targetApi="jelly_bean">@font/roboto_light</item>
```

Comment:
What sense in `tools:targetApi="jelly_bean"` if it's already lowest supported version?